src/__test__/dataparitytests/epochs.parity.test.ts

```diff
@@ -0,0 +1,170 @@
+import gql from 'graphql-tag'
+import { TestClient } from '../TestClient'
+import fetch from 'node-fetch'
+
+
+async function getDataFromAPI (path: string) {
```

Comment:
We're now repeating this definition, so it's best to create `util.ts` in the [upper scope](https://github.com/input-output-hk/cardano-graphql/tree/master/src/__test__), and then import into each test file. There's also a helper function here to extract and use: https://github.com/input-output-hk/cardano-graphql/blob/master/src/__test__/dataparitytests/transactions.parity.test.ts#L5-L12